### ISSUE TITLE
Undefined Symbol with Jazzy Packages

### ISSUE THREAD

**Issue by bochen87** [ros-perception/point_cloud_transport#97]
Hi,

I get the error of undefined symbol for jazzy release on an ARM computer `_ZN5draco17PointCloudBuilderC1Ev `

Something with the linking to draco lib seems wrong? Seems like it's not linked

```
$ldd /opt/ros/jazzy/lib/libdraco_point_cloud_transport.so
        linux-vdso.so.1 (0x0000ffffa00df000)
        libpoint_cloud_interfaces__rosidl_typesupport_cpp.so => /opt/ros/jazzy/lib/libpoint_cloud_interfaces__rosidl_typesupport_cpp.so (0x0000ffff9ffd0000)
        libpoint_cloud_transport.so => /opt/ros/jazzy/lib/aarch64-linux-gnu/libpoint_cloud_transport.so (0x0000ffff9fef0000)
        libclass_loader.so => /opt/ros/jazzy/lib/libclass_loader.so (0x0000ffff9fec0000)
        libconsole_bridge.so.1.0 => /usr/lib/aarch64-linux-gnu/libconsole_bridge.so.1.0 (0x0000ffff9fe90000)
        librclcpp.so => /opt/ros/jazzy/lib/librclcpp.so (0x0000ffff9fc20000)
        liblibstatistics_collector.so => /opt/ros/jazzy/lib/liblibstatistics_collector.so (0x0000ffff9fbf0000)
        librcl.so => /opt/ros/jazzy/lib/librcl.so (0x0000ffff9fb80000)
        libstatistics_msgs__rosidl_typesupport_cpp.so => /opt/ros/jazzy/lib/libstatistics_msgs__rosidl_typesupport_cpp.so (0x0000ffff9fb50000)
        libtracetools.so => /opt/ros/jazzy/lib/libtracetools.so (0x0000ffff9fb10000)
        librmw.so => /opt/ros/jazzy/lib/librmw.so (0x0000ffff9fae0000)
        librcutils.so => /opt/ros/jazzy/lib/librcutils.so (0x0000ffff9faa0000)
        libstdc++.so.6 => /usr/lib/aarch64-linux-gnu/libstdc++.so.6 (0x0000ffff9f820000)
        libgcc_s.so.1 => /usr/lib/aarch64-linux-gnu/libgcc_s.so.1 (0x0000ffff9f7e0000)
        libc.so.6 => /usr/lib/aarch64-linux-gnu/libc.so.6 (0x0000ffff9f620000)
        /lib/ld-linux-aarch64.so.1 (0x0000ffffa00a2000)
        libpoint_cloud_interfaces__rosidl_generator_c.so => /opt/ros/jazzy/lib/libpoint_cloud_interfaces__rosidl_generator_c.so (0x0000ffff9f5f0000)
        librosidl_typesupport_cpp.so => /opt/ros/jazzy/lib/librosidl_typesupport_cpp.so (0x0000ffff9f5c0000)
        libament_index_cpp.so => /opt/ros/jazzy/lib/libament_index_cpp.so (0x0000ffff9f590000)
        libtinyxml2.so.10 => /usr/lib/aarch64-linux-gnu/libtinyxml2.so.10 (0x0000ffff9f550000)
        libsensor_msgs__rosidl_typesupport_cpp.so => /opt/ros/jazzy/lib/libsensor_msgs__rosidl_typesupport_cpp.so (0x0000ffff9f520000)
        librcpputils.so => /opt/ros/jazzy/lib/librcpputils.so (0x0000ffff9f4f0000)
        librcl_interfaces__rosidl_typesupport_cpp.so => /opt/ros/jazzy/lib/librcl_interfaces__rosidl_typesupport_cpp.so (0x0000ffff9f4b0000)
        librcl_yaml_param_parser.so => /opt/ros/jazzy/lib/librcl_yaml_param_parser.so (0x0000ffff9f480000)
        librosgraph_msgs__rosidl_typesupport_cpp.so => /opt/ros/jazzy/lib/librosgraph_msgs__rosidl_typesupport_cpp.so (0x0000ffff9f450000)
        librcl_logging_interface.so => /opt/ros/jazzy/lib/librcl_logging_interface.so (0x0000ffff9f420000)
        librmw_implementation.so => /opt/ros/jazzy/lib/librmw_implementation.so (0x0000ffff9f3f0000)
        libtype_description_interfaces__rosidl_typesupport_c.so => /opt/ros/jazzy/lib/libtype_description_interfaces__rosidl_typesupport_c.so (0x0000ffff9f3c0000)
        librosidl_dynamic_typesupport.so => /opt/ros/jazzy/lib/librosidl_dynamic_typesupport.so (0x0000ffff9f380000)
        librosidl_typesupport_introspection_cpp.so => /opt/ros/jazzy/lib/librosidl_typesupport_introspection_cpp.so (0x0000ffff9f350000)
        librosidl_runtime_c.so => /opt/ros/jazzy/lib/librosidl_runtime_c.so (0x0000ffff9f310000)
        libm.so.6 => /usr/lib/aarch64-linux-gnu/libm.so.6 (0x0000ffff9f260000)
        librcl_logging_spdlog.so => /opt/ros/jazzy/lib/librcl_logging_spdlog.so (0x0000ffff9f230000)
        librcl_interfaces__rosidl_typesupport_c.so => /opt/ros/jazzy/lib/librcl_interfaces__rosidl_typesupport_c.so (0x0000ffff9f200000)
        libtype_description_interfaces__rosidl_generator_c.so => /opt/ros/jazzy/lib/libtype_description_interfaces__rosidl_generator_c.so (0x0000ffff9f1c0000)
        libyaml-0.so.2 => /usr/lib/aarch64-linux-gnu/libyaml-0.so.2 (0x0000ffff9f180000)
        libstatistics_msgs__rosidl_generator_c.so => /opt/ros/jazzy/lib/libstatistics_msgs__rosidl_generator_c.so (0x0000ffff9f150000)
        liblttng-ust.so.1 => /usr/lib/aarch64-linux-gnu/liblttng-ust.so.1 (0x0000ffff9f0b0000)
        libsensor_msgs__rosidl_generator_c.so => /opt/ros/jazzy/lib/libsensor_msgs__rosidl_generator_c.so (0x0000ffff9f040000)
        libstd_msgs__rosidl_generator_c.so => /opt/ros/jazzy/lib/libstd_msgs__rosidl_generator_c.so (0x0000ffff9eff0000)
        libbuiltin_interfaces__rosidl_generator_c.so => /opt/ros/jazzy/lib/libbuiltin_interfaces__rosidl_generator_c.so (0x0000ffff9efc0000)
        librcl_interfaces__rosidl_generator_c.so => /opt/ros/jazzy/lib/librcl_interfaces__rosidl_generator_c.so (0x0000ffff9ef50000)
        librosgraph_msgs__rosidl_generator_c.so => /opt/ros/jazzy/lib/librosgraph_msgs__rosidl_generator_c.so (0x0000ffff9ef20000)
        librosidl_typesupport_c.so => /opt/ros/jazzy/lib/librosidl_typesupport_c.so (0x0000ffff9eef0000)
        libspdlog.so.1.12 => /usr/lib/aarch64-linux-gnu/libspdlog.so.1.12 (0x0000ffff9ee70000)
        libservice_msgs__rosidl_generator_c.so => /opt/ros/jazzy/lib/libservice_msgs__rosidl_generator_c.so (0x0000ffff9ee40000)
        libnuma.so.1 => /usr/lib/aarch64-linux-gnu/libnuma.so.1 (0x0000ffff9ee10000)
        liblttng-ust-common.so.1 => /usr/lib/aarch64-linux-gnu/liblttng-ust-common.so.1 (0x0000ffff9ede0000)
        liblttng-ust-tracepoint.so.1 => /usr/lib/aarch64-linux-gnu/liblttng-ust-tracepoint.so.1 (0x0000ffff9eda0000)
        libgeometry_msgs__rosidl_generator_c.so => /opt/ros/jazzy/lib/libgeometry_msgs__rosidl_generator_c.so (0x0000ffff9ed40000)
        libfmt.so.9 => /usr/lib/aarch64-linux-gnu/libfmt.so.9 (0x0000ffff9ed00000)
```

**Comment by bochen87** [comment]
Somehow it seems `DRACO_LIBRARY` has been changed to `DRACO_LIBRARIES` - i was able to successfully link and run it with that change

**Comment by bochen87** [comment]
resolved with [#57](https://github.com/ros-perception/point_cloud_transport_plugins/pull/57)
